sdk/core/ci.yml

```diff
@@ -42,6 +42,45 @@ stages:
       LiveTestTimeoutInMinutes: 90 # default is 60 min. We need a little longer on worst case for Win+jsonTests
       LineCoverageTarget: 93
       BranchCoverageTarget: 55
+      PreTestSteps:
+        - pwsh: |
+            docker build -t squid-local $(Build.SourcesDirectory)/sdk/core/azure-core/test/ut/proxy_tests/localproxy
+            docker build -t squid-local.passwd $(Build.SourcesDirectory)/sdk/core/azure-core/test/ut/proxy_tests/localproxy.passwd
+            docker run --rm -d -p 3128:3128 squid-local
+            docker run --rm -d -p 3129:3129 squid-local.passwd
```

Comment:
Looks like these lines are similar to `runproxy.ps1` and `builddocker.ps1`. If we expect to modify these in the future, should both places be changed and remain identical/in-sync?

Reply:
I'm seeing if I can do the runproxy script in ci.yml.